language/stdlib/modules/DiemAccount.move

```diff
@@ -440,28 +440,27 @@ module DiemAccount {
     public fun cancel_burn<Token>(
         account: &signer,
         preburn_address: address,
+        amount: u64,
     ) acquires DiemAccount, Balance, AccountOperationsCapability {
-        let coin = Diem::cancel_burn<Token>(account, preburn_address);
+        let coin = Diem::cancel_burn<Token>(account, preburn_address, amount);
```

Comment:
This is the function that is timing out. 

If I add a `spec { assume false; }` after this line `DiemAccount` verifies. However if I remove it this function times out. 

I thought it was due to the fact that the `value` of the returned coin from `Diem::cancel_burn` wasn't specified and this was causing the `ensures balance_at_addr` condition to fail, but even after adding that to the spec for `Diem::cancel_burn` we still get this timeout.